services/frontend-v3/src/components/layouts/appLayout/AppLayoutView.jsx

```diff
@@ -21,35 +21,37 @@ const AppLayoutView = ({
   const { locale } = useSelector((state) => state.settings);
 
   return (
-    <Layout style={{ minHeight: "100vh" }}>
-      <Helmet>
-        <html lang={locale === "ENGLISH" ? "en" : "fr"} />
-      </Helmet>
-      <TopNav
-        loading={loading}
-        displayLogo={displayLogo}
-        displaySearch={displaySearch}
-      />
-      <Layout>
-        <SideNav
-          sideBarContent={sideBarContent}
-          displaySideBar={displaySideBar}
+    <>
+      <Layout style={{ minHeight: "100vh" }}>
+        <Helmet>
+          <html lang={locale === "ENGLISH" ? "en" : "fr"} />
+        </Helmet>
+        <TopNav
           loading={loading}
+          displayLogo={displayLogo}
+          displaySearch={displaySearch}
         />
         <Layout>
-          <Content className="app-content">
-            {loading ? (
-              <Card>
-                <Skeleton active />
-              </Card>
-            ) : (
-              children
-            )}
-          </Content>
-          <Footer />
+          <SideNav
+            sideBarContent={sideBarContent}
+            displaySideBar={displaySideBar}
+            loading={loading}
+          />
+          <Layout>
+            <Content className="app-content">
+              {loading ? (
+                <Card>
+                  <Skeleton active />
+                </Card>
+              ) : (
+                children
+              )}
+            </Content>
+            <Footer />
+          </Layout>
         </Layout>
       </Layout>
-    </Layout>
+    </>
   );
 };
 
```

Comment:
I don't understand the change in this file, is it just because we were planning on putting the modal in the AppLayoutView? (should we revert the changes to this file)

Reply:
resolved